[2023-04-02 11:38:55,301] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_and_backfill_v02.first_task 2023-04-01T00:00:00+00:00 [queued]>
[2023-04-02 11:38:55,370] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_and_backfill_v02.first_task 2023-04-01T00:00:00+00:00 [queued]>
[2023-04-02 11:38:55,372] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-04-02 11:38:55,376] {taskinstance.py:1069} INFO - Starting attempt 1 of 6
[2023-04-02 11:38:55,376] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-04-02 11:38:55,444] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): first_task> on 2023-04-01T00:00:00+00:00
[2023-04-02 11:38:55,468] {standard_task_runner.py:52} INFO - Started process 50 to run task
[2023-04-02 11:38:55,479] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_catchup_and_backfill_v02', 'first_task', '2023-04-01T00:00:00+00:00', '--job-id', '74', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cathcup_and_backfill.py', '--cfg-path', '/tmp/tmpnszepz6c', '--error-file', '/tmp/tmpgahap9_m']
[2023-04-02 11:38:55,488] {standard_task_runner.py:77} INFO - Job 74: Subtask first_task
[2023-04-02 11:38:55,836] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v02.first_task 2023-04-01T00:00:00+00:00 [running]> on host 8897c098f7c0
[2023-04-02 11:38:56,090] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mario
AIRFLOW_CTX_DAG_ID=dag_with_catchup_and_backfill_v02
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2023-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-01T00:00:00+00:00
[2023-04-02 11:38:56,100] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-04-02 11:38:56,104] {bash.py:158} INFO - Running command: echo catchup on Airflow!!!
[2023-04-02 11:38:56,189] {bash.py:169} INFO - Output:
[2023-04-02 11:38:56,448] {bash.py:173} INFO - catchup on Airflow!!!
[2023-04-02 11:38:56,544] {bash.py:177} INFO - Command exited with return code 0
[2023-04-02 11:38:56,823] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v02, task_id=first_task, execution_date=20230401T000000, start_date=20230402T113855, end_date=20230402T113856
[2023-04-02 11:38:56,987] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-04-02 11:38:57,241] {local_task_job.py:146} INFO - Task exited with return code 0
